sequencing/ATACSeq/config/config.txt

```diff
@@ -1,41 +1,32 @@
-## ===========================================================##
-##           ATAC-seq pipeline: Example config file           ##
-## ===========================================================##
-##                                                            ||
-## This is an example config file for the ATAC-sequencing     ||
-## pipeline. This is expected to be located in the project's  ||
-## directory. The full path to this file should be specified  ||
-## when running the pipeline's scripts.                       ||
-##                                                            ||
-## The project's folder name needs to be specified in the     ||
-## first variable. Please specify the modules versions to be  ||
-## used.                                                      ||
-##                                                            ||
-## ===========================================================##
-
-
-## Example config file ##
-## This config file should be found at the project's folder. The full path to this file should be specified 
-## Please modify as required ##
-
-## Sequencing data filepaths
+## ===========================================================================##
+##                      ATAC-seq pipeline: Example config file                ##
+## ===========================================================================##
+##                                                                            ||
+## This is an example config file for the ATAC-sequencing pipeline.           ||
+## This file and the R config file are expected to be located in the project's||
+## directory.                                                                 ||
+## The full path to this file should be specified as the first command line   ||
+## argument when running the pipeline's scripts.                              ||
+##                                                                            ||
+## ===========================================================================##
+
+## ========================= ##
+|| Project set up filepaths  ||
+## ========================= ##
+
+## Specify the name of the project's directory ##
 export PROJECT="project name"
+## Specify the full project's directory ##
 export MAIN_DIR="/path/to/project/folder/${PROJECT}"
```

Comment:
Yeah this works well. Separating it out is probably the best solution

Reply:
I will add extra info in the README file to make very clear what needs to be changed and what datasets, references, etc are expected from the user

Reply:
good idea